.gitignore

```diff
@@ -22,4 +22,14 @@
 .DS_Store
 *.bkp
 .python-version
-vendor/
\ No newline at end of file
+vendor/
+
+# poetry and flutter lock files
+*.lock
```

Comment:
Not sure about ignoring `.lock` files. We definitely want to add Poetry and Flutter locks into repository. Please remove for ignores.